velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <hdfs/hdfs.h>
```

Comment:
We used the PIMPL idiom for S3FileSystem to avoid including a third-party header in this header. PIMPL helps keep the API clean from third-party headers.

Reply:
Understood. Will do.